sdk/core/azure-core/inc/azure/core/internal/tracing/service_tracing.hpp

```diff
@@ -4,7 +4,8 @@
 #include "azure/core/context.hpp"
 #include "azure/core/internal/client_options.hpp"
 #include "azure/core/internal/http/user_agent.hpp"
-#include "azure/core/tracing/tracing.hpp"
+#include "azure/core/internal/tracing/tracing_impl.hpp"
```

Comment:
```suggestion
#include "azure/core/internal/tracing/tracing_impl.hpp"

```

Reply:
(new line before stdlib include)

Reply:
I prefer that there NOT be newlines in the #includes - newlines function as context breaks for clang-format and IMHO should only ever be used when you absolutely need to specify the order of header inclusion.

So why should we have a break between the standard headers and the internal headers?

Reply:
To separate vsually, I think all other code has it, but also from practical standpoint so that after applying auto-format, include section does not look like

```
#include <algorithm>
#include <azure/...>
#include <vector>
```